src/MeetingsSDKAdapter.test.js

```diff
@@ -859,57 +858,38 @@ describe('Meetings SDK Adapter', () => {
     });
   });
 
-  describe('switchCameraControl()', () => {
-    test('returns the display data of a meeting control in a proper shape', (done) => {
-      meetingsSDKAdapter.switchCameraControl(meetingID)
-        .pipe(take(1)).subscribe((dataDisplay) => {
-          expect(dataDisplay).toMatchObject({
-            ID: 'switch-camera',
-            tooltip: 'Video Devices',
-            options: null,
-            selected: null,
-          });
-          done();
-        });
-    });
-
-    test('throws errors if sdk meeting object is not defined', (done) => {
-      meetingsSDKAdapter.fetchMeeting = jest.fn();
+  describe('switchCamera()', () => {
+    test('sets the camera that was chosen by the user', async () => {
+      meetingsSDKAdapter.meetings[meetingID].cameraID = null;
+      await meetingsSDKAdapter.switchCamera(meetingID, 'cameraID');
```

Comment:
I would use something like `'example-camera-id'` in tests because `cameraID` looks like a field name, not value